aspects/aspects.go

```diff
@@ -37,26 +37,21 @@ import (
 type accessType int
 
 const (
-	readWrite accessType = iota
+	readWrite accessType = 1 << iota
```

Comment:
why this change?

Reply:
I think it might've been to simplify the check from a switch to `rule.access&(read|readWrite) != 0`. But reading it now, it seems unnecessary